details/operator-sdk/api/v1/astraconnector_types.go

```diff
@@ -9,85 +9,87 @@ import (
 )
 
 type Astra struct {
-	Unregister  bool   `json:"unregister,omitempty"`
-	Token       string `json:"token,omitempty"`
-	ClusterName string `json:"clusterName,omitempty"`
-	AccountID   string `json:"accountId"`
+	AccountId string `json:"accountId"`
 	// +kubebuilder:validation:Optional
-	CloudID string `json:"cloudID"`
+	CloudId string `json:"cloudId"`
 	// +kubebuilder:validation:Optional
-	ClusterID string `json:"clusterID"`
+	ClusterId   string `json:"clusterId"`
+	ClusterName string `json:"clusterName,omitempty"`
 	// +kubebuilder:validation:Optional
 	StorageClassName  string `json:"storageClassName"`
 	SkipTLSValidation bool   `json:"skipTLSValidation,omitempty"`
+	Token             string `json:"token,omitempty"`
```

Comment:
I think we had agreed to calling this `tokenReference` just as an indication that it's a secret instead of the token directly.